### DISCUSSION TITLE
Ai協助判斷憂鬱症患者嚴重程度

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#744] [General]
有團隊開始研發當著醫生進行診斷的ai機器人
幫助醫生進行更專業與精確的判斷
而目前最常見憂鬱症患者的治療方式為藥物治療、音樂治療和心理諮商治療
我想如果可以訓練機器人 進行輕微程度的心理諮商治療
說不定會是個醫療轉機
如此不僅可以減輕醫生的負擔
身為患者可能也因為面對的醫生非真人
而更願意敞開心胸去談話
https://www.bnext.com.tw/article/52971/compal-invest-hippo-screen-ai-depression-help

**Comment by alvin1226** [comment]
這會不會有誤判的可能性呢？

**Comment by ghost** [comment]
雖然ai幫助人判斷疾病已經屢見不鮮，但是我們必須思考這種有關人類心理方面的問題，是否也可以透過ai這種冷冰冰的科技產物來得到治療。

**Comment by yjpxi** [comment]
我覺得ai機器人的好處是可以隨時都可以用，不像醫院的醫生都有時間限制，然而壞處是ai會缺乏靈活性，有可能沒辦法處理複雜或個性化的情境。
